'Our Tours' page changes (#272)

manika137 opened the pull request.

Arun9739 commented:
@manika137 Plz follow the contributing guidelines
Add the issue number, changes you made and the description of changes, checklist, screenshot etc. (which were there at original PR format)

manika137 commented:
## Related Issue
Issue https://github.com/Arun9739/Paryatana/issues/272#issue-1410176253

## Changes you made :
UI Enhancement and fixed bugs on the 'Our Tours' page

## Describe the changes :
- Added space between the slider and next section
- Made the touring boxes of uniform size
- Corrected spellings (Rajasthan, Meghalaya etc)

## Check list :
- [x] I have followed code style of this project.
- [x] I have read the [CONTRIBUTING](https://github.com/Arun9739/Paryatana/blob/main/Contributing.md) document.
- [x] All the aspects mentioned in the issue are covered.
- [x]  The website is properly working after my changes.
- [x]🌟 ed the repo

## Type of change :
 - [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Output Screenshots :
<img width="949" alt="image" src="https://user-images.githubusercontent.com/80244229/196133685-e33d80c3-c27c-466a-88bf-9d8c39e7dda4.png">
<img width="947" alt="image" src="https://user-images.githubusercontent.com/80244229/196133786-87f30354-717f-42f3-9e67-120c696135b1.png">

Arun9739 commented:
@manika137 Nice 🤩 
LGTM
